### DISCUSSION TITLE
using undrift by aim in batch postprocessing

### DISCUSSION THREAD

**Discussion by ICheSysBio** [jungmannlab/picasso#505] [Q&A] [answered]
Hi all,

Are there ways to apply AIM in batch postprocessing or without using the render gui? 

I have tried to use the AIM in batch postprocessing from cmd. Based on the undrift options (mode & segmentation) I assume that AIM is not callable from cmd at the moment (v0.7.3).

Cheers,
Abarto

**Comment by rafalkowalewski1** [comment, marked as answer]
Hi Abarto, that's a very good point, I will add this option in the next release.

For now, if you're okay with running picasso from another python script (not directly from the command window), please use the function picasso.aim.aim: https://github.com/jungmannlab/picasso/blob/14ce66cb1a7dad9777a32592d71dc6ba493db482/picasso/aim.py#L607.

Bests,
Rafal
